docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -279,6 +281,18 @@ The converter can deserialize JSON that was created by using the same converter
 
 The converter code in the preceding example reads and writes each property manually. An alternative is to call `Deserialize` or `Serialize` to do some of the work. For an example, see [this StackOverflow post](https://stackoverflow.com/a/59744873/12509023).
 
+### Support round-trip for Stack\<T>
+
+If you deserialize a JSON string into a <xref:System.Collections.Generic.Stack%601> object and then serialize that object, the contents of the stack are in reverse order. To support serialization and deserialization that retains the original order in the stack, a custom converter is required.
```

Comment:
From the issue description (https://github.com/dotnet/docs/issues/16690), should we mention all the types that this impacts (including types that derive from them)?

@layomia, is this list still accurate?
```text
This behavior applies to the following types and interface and user-defined types that
derive from them:	
* <xref:System.Collections.Stack>	
* <xref:System.Collections.Generic.Stack%601>	
* <xref:System.Collections.Immutable.ImmutableStack%601>	
* <xref:System.Collections.Immutable.IImmutableStack%601>	
```

Reply:
also `ConcurrentStack<T>`.